tools/checklocks/checklocks.go

```diff
@@ -138,8 +167,18 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			continue
 		}
 
+		var start time.Time
+		if Benchmark {
+			start = time.Now()
```

Comment:
Add start and end as fields to PerfData. Less stuff interacting, more information, and low cost for storage.